common/unsafe/src/main/java/org/apache/spark/sql/catalyst/expressions/HiveHasher.java

```diff
@@ -38,6 +39,10 @@ public static int hashLong(long input) {
     return (int) ((input >>> 32) ^ input);
   }
 
+  public static int hashUnsafeBytesMB(MemoryBlock base, long offset, int lengthInBytes) {
```

Comment:
`BytesMB` seemed weird to read as `MB` is generally interpreted as mega bytes. How about `hashUnsafeBlock` ?